frameworks/bf/run-bf.js

```diff
@@ -0,0 +1,7 @@
+var execSync = require('child_process').execSync;
+module.exports.runBf = function(input) {
```

Comment:
Try adding option `encoding: 'utf8'` for `execSync`, or just call `.toString()` on returned buffer.